service/tbc/tbc.go

```diff
@@ -913,33 +1033,39 @@ func (s *Server) handleAddrV2(_ context.Context, p *peer, msg *wire.MsgAddrV2) {
 	}
 
 	if err := s.pm.PeersInsert(peers); err != nil {
-		log.Errorf("Insert peers: %v", err)
+		return fmt.Errorf("Insert peers: %w", err)
```

Comment:
Error strings shouldn't start with capitals, per https://go.dev/wiki/CodeReviewComments#error-strings
```suggestion
		return fmt.Errorf("insert peers: %w", err)
```